docs/getting_started.md

```diff
@@ -11,6 +11,8 @@
 
 ## 1. Install Argo CD
 
+### Normal Install
```

Comment:
Will be happy to replace 'Normal' with some other term... any suggestion is greatly appreciated!

Reply:
I propose either "Default" or "Local" install.